toc.json

```
@@ -45,17 +59,12 @@
     },
     {
       "type": "divider",
-      "title": "Balances"
+      "title": "Cash savings"
     },
     {
       "type": "item",
-      "title": "Basics",
-      "uri": "docs/balances/Basics.md"
-    },
-    {
-      "type": "item",
-      "title": "FAQs",
-      "uri": "docs/balances/FAQs.md"
+      "title": "Overview",
```

```suggestion
      "title": "Basics",
```
"Basics" is more inline with other sections.